csaf/src/rolie/roliefeed.rs

```diff
@@ -0,0 +1,71 @@
+use serde::{Deserialize, Serialize};
+use time::OffsetDateTime;
+
+#[derive(Serialize, Deserialize)]
```

Comment:
For this and similar structs, please add `Debug, Clone, PartialEq` and also `Eq` if possible.